articles/cosmos-db/indexing-policies.md

```diff
@@ -156,7 +156,7 @@ Now that we've taken a look at how to specify paths, let's look at the options w
 
 * Data type: **String**, **Number**, **Point**, **Polygon**, or **LineString** (can contain only one entry per data type per path)
 * Index kind: **Hash** (equality queries), **Range** (equality, range or Order By queries), or **Spatial** (spatial queries) 
-* Precision: 1-8 or -1 (Maximum precision) for numbers, 1-100 (Maximum precision) for string
+* Precision: For hash index this varies from 1 to 8 for both strings and numbers with default as 3. For range index this value can be -1 (Maximum precision) and vary between 1-100 (Maximum precision) for string or number values.
```

Comment:
Govind - can you please edit as such: For hash index, this varies from 1 to 8 for both strings and numbers with a default value of 3. For range index, this value can be set to -1 (for maximum precision) and vary between 1-100 (maximum precision) for string or number values. 

However, it doesn't make sense just in reading this why -1 and 1-100 are both "maximum precision". Is that a typo or correct?